components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp

```diff
@@ -105,14 +104,14 @@
 					<div class="control-group">
 					   <% if (loginFailed) { %>
                           <div class="alert alert-error">
-                              <%=errorMessage%>
+                              <%=Encode.forHtmlContent(errorMessage)%>
```

Comment:
Use Encode.forHtml here since we can embed HTML tags within <div>. So it's important to encode for all significant HTML characters